controllers/instanaagent_controller.go

```diff
@@ -389,7 +471,14 @@ func (r *InstanaAgentReconciler) updateStatusFields(ctx context.Context, req ctr
 	if !cmp.Equal(configMapResource, crdInstance.Status.ConfigMap) ||
 		!cmp.Equal(daemonSetResource, crdInstance.Status.DaemonSet) {
 
-		return r.upsertCrdStatusFields(ctx, req, func(status *instanaV1.InstanaAgentStatus) instanaV1.InstanaAgentStatus {
+		r.recorder.Event(crdInstance, "Normal", "CreatedUpdated", "The Agent deployment was correctly created / updated")
```

Comment:
Next to the error events, making sure there's also a success event so that customers can see everything should be correct after a potential failure.